lib/controllers/cakes.js

```diff
@@ -0,0 +1,13 @@
+import { Router } from 'express';
+import Cake from '../models/Cake';
+
+export default Router()
+  .post('/', async (req, res, next) => {
+    try {
+      const cake = await Cake.insert(req.body);
```

Comment:
Your post route will probably want to call your `CakeService.createCake()` method, rather than `Cake.insert()` so that it will do the Twilio things you've included as well as insert a new cake into the database.